extra-network/zerotier-one/autobuild/build

```diff
@@ -4,6 +4,7 @@ sed -e 's/sbin/bin/' \
     -i "$SRCDIR"/debian/zerotier-one.service
 
 abinfo "Making zerotier-one..."
+export ZT_DEBUG=1
```

Comment:
Explain what this does.

Reply:
I haven't found a way to compile it successfully in autobuild3 without turn on DEBUG option. It may related to some compile flags but I can't see which. Without DEBUG flag, the error should be `probestack` undefined. However, I need some help.